barretenberg/cpp/src/barretenberg/bb/get_bn254_crs.cpp

```diff
@@ -22,6 +22,7 @@ std::vector<uint8_t> download_bn254_g1_data(size_t num_points)
 std::vector<uint8_t> download_bn254_g2_data()
 {
     std::string url = "https://aztec-ignition.s3.amazonaws.com/MAIN%20IGNITION/flat/g2.dat";
+    // IMPORTANT: this currently uses a shell, DO NOT let user-controlled strings here.
```

Comment:
Note: this change is along for the ride as I didn't feel like making a separate PR